cves/CVE-2012-0858.yml

```diff
@@ -313,8 +316,9 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: The fix for this is seperating the pointers for memory, so if any of the buffer are invalid 
+      or there is an error the free still works.
```

Comment:
Does this actually have to do with the input that the user of the system would be giving, ie is the invalid free a direct result of the user input? If not, then I believe that 'distrust inputs' would be false here.

Reply:
Improved my description to better say why I marked it as true